src/components/navigation/navigation.js

```diff
@@ -45,15 +62,16 @@ function NavBar() {
     return (
       <div className="App-Nav">
         <div className="img-container">
-          <img
-            alt="Express Groomer Logo"
-            src="./images/express_groomer.jpg"
-            width="200px"
-            height="85px"
-          />
+          <div class="express-logo">
+            <span class="groomer-one">Express</span>{' '}
+            <span class="groomer-two">Groomer</span>
+          </div>
```

Comment:
When I pulled these changes down to my local environment, using class here instead of className created a warning.

Reply:
Fixing that now!